spec/i18n_spec.rb

```diff
@@ -149,12 +149,6 @@ def allowed_untranslated_key?(locale, key)
         expect(bad_keys).to be_empty
       end
 
-      it 'is formatted as normalized YAML' do
-        normalized_yaml = YAML.dump(YamlNormalizer.handle_hash(YAML.load_file(full_path)))
-
-        expect(File.read(full_path)).to(eq(normalized_yaml), 'run `make normalize_yaml` to fix')
-      end
```

Comment:
just confirming, the result of the test will still be run because of the new `make lint_yaml` step

Reply:
Yeah, basically trying to avoid this scenario in the future by ensuring that `config/country_dialing_codes.yml` is included in this testing as well.

Reply:
Apparently I'd missed this file!

https://github.com/18F/identity-idp/blob/main/spec/config/country_dialing_codes.yml_spec.rb

But with the updated Make task, I assume it'd be safe to remove this.

Reply:
>But with the updated Make task, I assume it'd be safe to remove this.

Removed in 2578baa.